src/components/artwork_filter/headline.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from "react"
+
+import { secondary } from "../../assets/fonts"
+import labelMap from "./param_map"
+
+import { compact } from "lodash"
+import styled from "styled-components"
+
+interface HeadlineProps extends React.HTMLProps<Headline> {
+  medium: any,
```

Comment:
Could you assign types to these properties instead of `any`? It'll make the typescript compiler a lot of more helpful with type checking and autocompletion